### PR TITLE
feat: add fasttrack link to webui

### LINKED ISSUE #1919: Let's add a button for Backend.AI FastTrack if supported
### Main idea

> ⚠️ NOTE: This issue is suggested by customers.

Since tasks in the pipeline created from Backend.AI FastTrack are reflected in sessions in webUI respectively, It's better to provide a single button to access Backend.AI FastTrack site. But This feature is enabled only if they support Backend.AI FastTrack internally.

### Alternative ideas

_No response_

### Anything else?

_No response_
```[tasklist]
### Tasks
- [ ] https://github.com/lablup/backend.ai-webui/pull/1929
- [ ] https://github.com/lablup/backend.ai/pull/1576
```

### PR DESCRIPTION
**Pull request by agatha197**
resolves #1919 
add fasttrack link to the footer of summary page if endpoint exists on config.toml.
counterpart PR: https://github.com/lablup/backend.ai/pull/1576

**Updated**
image

mini-ui
image

### CODE REVIEW

**Review by adrysn — COMMENTED** [review]
The location of the "FastTrack" link seems to be hard to find. This is just an opinion, but how about add the icon at the bottom part of the sidebar just beside the documentation link with a tooltip text?

image

File: src/components/backend-ai-summary-view.ts [hunk 1]
```diff
@@ -389,6 +390,8 @@ export default class BackendAISummary extends BackendAIPage {
           this.webui_version = globalThis.packageVersion;
           this.appDownloadUrl =
             globalThis.backendaiclient._config.appDownloadUrl;
+          this.fasttrackEndpoint =
+            globalThis.backendaiclient._config.fasttrack_endpoint;
```
Comment (inureyes) [inline src/components/backend-ai-summary-view.ts]: @agatha197 Please check `fasttrack_endpoint` to `CamelCase` format like others.

File: src/components/backend-ai-login.ts [hunk 2]
```diff
@@ -936,6 +938,20 @@ export default class BackendAILogin extends BackendAIPage {
...
   }
 
+  /**
+   * Initialize global key with value from pipeline section in config file
+   *
+   * @param {object} pipelineConfig
+   */
+  private _initPipelineConfigWithKeys(pipelineConfig) {
+    // FastTrack endpoint
+    this.fasttrack_endpoint = this._getConfigValueByExists(pipelineConfig, {
```
Comment (inureyes) [inline src/components/backend-ai-login.ts]: @agatha197 Please check fasttrack_endpoint to CamelCase format like others.

**Review by inureyes — CHANGES_REQUESTED** [review]
@agatha197 Please check fasttrack_endpoint to CamelCase format like others.

**Comment by inureyes** [comment]
> The location of the "FastTrack" link seems to be hard to find. This is just an opinion, but how about add the icon at the bottom part of the sidebar just beside the documentation link with a tooltip text?
> 
> image

It is folded when using thin sidebar mode.

**Comment by agatha197** [comment]
@yomybaby suggested adding fasttrack link to the side menu and displaying it with an external icon like this. 

How about this @adrysn ?

File: src/components/backend-ai-summary-view.ts (hunk 1 above)
Comment (agatha197) [reply src/components/backend-ai-summary-view.ts]: reflected!

**Comment by adrysn** [comment]
> @yomybaby suggested adding fasttrack link to the side menu and displaying it with an external icon like this. image How about this @adrysn ?

I'm okay with that.

**Comment by yomybaby** [comment]
LGTM.
Next time, let's make it so that the admin can add external links to the side menu using icon, title, and link options.

**Review by yomybaby — APPROVED** [review]

**Review by yomybaby — APPROVED** [review]

**Pull request merged**